pkg/storagecapabilities/storagecapabilities.go

```diff
@@ -31,7 +31,9 @@ var CapabilitiesByProvisionerKey = map[string][]StorageCapabilities{
 		{AccessMode: v1.ReadWriteOnce, VolumeMode: v1.PersistentVolumeBlock},
 		{AccessMode: v1.ReadWriteOnce, VolumeMode: v1.PersistentVolumeFilesystem}},
 	"openshift-storage.rbd.csi.ceph.com": {
-		{AccessMode: v1.ReadWriteMany, VolumeMode: v1.PersistentVolumeBlock}},
+		{AccessMode: v1.ReadWriteMany, VolumeMode: v1.PersistentVolumeBlock},
+		{AccessMode: v1.ReadWriteOnce, VolumeMode: v1.PersistentVolumeBlock},
+		{AccessMode: v1.ReadWriteOnce, VolumeMode: v1.PersistentVolumeFilesystem}},
```

Comment:
Caps for all 4 ceph-rbd keys are the same, so to make it more readable amd maintainable, consider using an initializer func, e.g.
```
func getCephRbdCaps() []StorageCapabilities {
	return []StorageCapabilities{
		{AccessMode: v1.ReadWriteMany, VolumeMode: v1.PersistentVolumeBlock},
		{AccessMode: v1.ReadWriteOnce, VolumeMode: v1.PersistentVolumeBlock},
		{AccessMode: v1.ReadWriteOnce, VolumeMode: v1.PersistentVolumeFilesystem},
	}
}
```
and similarly: `getRWOFilesystemCaps, getRWXFilesystemCaps, getRWOBlockCaps`

Reply:
I would like to get another vote on that. In my opinion it is clearer the way it is. We have an explicit table mapping the key (based on provider string) to capabilities. Almost looks like a spreadsheet.

Reply:
Spreadsheets were invented mostly for using functions instead of c&p'ing data across the cells.